stl/src/stacktrace.cpp

```diff
@@ -78,7 +78,7 @@ namespace {
             initialize_attempted = false;
         }
 
-        [[nodiscard]] bool try_initialize() noexcept {
+        [[nodiscard]] static bool try_initialize() noexcept {

```

Comment:
These are intentionally nob-static to ensure synchronization, please revert adding `static` in this file